network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/VoltageLevelImpl.java

```diff
@@ -386,6 +386,8 @@ public int getConnectableCount() {
             return (List<T>) getDanglingLines();
         } else if (clazz == Line.class) {
             return (List<T>) index.getLines(resource.getId());
+        } else if ((clazz == BusbarSection.class) && resource.getAttributes().getTopologyKind() == TopologyKind.NODE_BREAKER) {
+            return (List<T>) index.getBusbarSections(resource.getId());
         }
```

Comment:
In case TopologyKind is BUS_BREAKER, maybe throw a specific exception (to avoid the TODO exception to be thrown)

Reply:
Done